src/app/app.module.ts

```diff
@@ -41,24 +47,27 @@ import {SideMenuCompModule} from "./components/sideMenu/sideMenu.module";
     AngularFireAuthModule,
     SharedModule.forRoot(),
     PeerToInstructorCardModule,
+    BoardCardModule,
     P2iNewQuestionModule,
     SignInPageModule,
-    DashboardPageModule,
     SideMenuCompModule
   ],
   bootstrap: [IonicApp],
   entryComponents: [
     MyApp,
+    DashboardPage,
     InstructorFeedPage,
     StudentFeedPage,
     SortPopOverComponent,
-    PeerToInstructorCardComponent
+    PeerToInstructorCardComponent,
+    BoardCardComponent
   ],
   providers: [
     StatusBar,
     SplashScreen,
     {provide: ErrorHandler, useClass: IonicErrorHandler},
-    PopOverSortCommService
+    PopOverSortCommService,
+    BoardService
```

Comment:
I'm confused that we have declared provider of board service in component ts file. Why we're declaring it again here and in shared.module ?

Reply:
Oh I'm not sure. Maybe we dont need it.